### PR TITLE
Check Readme for details about this PR.

### PR DESCRIPTION
**Pull request by shoukoo**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,12 @@
+
+ver:
+	@read -p "Enter a new version:" version; \
+	old_version=$(shell grep -o "\d.\d.\d" Dockerfile);  \
+	echo "$$old_version to $$version"; \
+	for i in $(shell find . -name "Dockerfile"); do sed -i "s/$$old_version/$$version/g" $$i; done;
```
Comment (alexlance) [inline Makefile:6]: Not sure if this is helpful, but Dockerfiles can contain arguments that are passed into the `docker build` command. That might be a bit more elegent than running `sed -i` on the file.

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+sync:
+	cat setup.sh | tee $(shell find . -name "gp-setup.sh")
```
Comment (alexlance) [inline Makefile:9]: This might be a tiny bit easier to read:
```cp setup.sh $(shell find . -name "gp-setup.sh")```

File: README.md [hunk 3]
```diff
@@ -1,2 +1,98 @@
 # golang-pipeline
-Github action for Go, it tests and builds your code seamlessly without any hassle.
+
+## Quick Install
+Run the below command to create workflows in your repo.
+```bash
+curl
```
Comment (alexlance) [inline README.md]: This might be missing the example? ^^

**Review by alexlance — APPROVED** [review]
Woah, you've been busy :)

File: Makefile (hunk 2 above)
Comment (shoukoo) [reply Makefile:9]: that doesn't work,  I don't think `cp` command can copy to multiple targets.

File: Makefile (hunk 1 above)
Comment (shoukoo) [reply Makefile:6]: Cool, Github Action calls docker build in the background and developers don't have the control to modify the command.

File: README.md (hunk 3 above)
Comment (shoukoo) [reply README.md]: Will add it shortly

**Pull request merged**